### PR TITLE
address is always required

### PR DESCRIPTION
**Pull request by nuclearWinters**

### CODE REVIEW

**Review by r9software — APPROVED** [review]

File: src/components/billing-info-container/index.tsx [hunk 1]
```diff
@@ -635,6 +635,9 @@ export const BillingInfoContainer = ({
                                 if (el.name === 'phone') {
                                   el.required = flagRequirePhone
                                 }
+                                if (el.name === 'street_address') {
```
Comment (ft-worker) [inline src/components/billing-info-container/index.tsx:638]: @nuclearWinters this is wrong approach. There is `billingInfoFields` list of objects where we defined which field is required and which one is optional. Here we can modify field attributes if they depend on another flag or value (see line 636).

**Comment by ft-worker** [comment]
These changes are unnecessary and they need to be reverted. `street_address` field must be controlled from outside.

**Comment by purplelight** [comment]
@ft-worker the production house of x site was broken due to some address issues and it caused a massive problem between us and the client. Can you make sure what you're doing does not affect that?

**Comment by ft-worker** [comment]
@purplelight new PR will be opened soon. The only reason I've reverted this PR, is that changes need to be done in HOX's repo. Please note that there is no auto-deploy and current changes won't have affect yet.

**Comment by purplelight** [comment]
@ft-worker Ok thanks. We must make sure nothing breaks on the HoX site

**Pull request merged**